src/App.js

```diff
@@ -1,16 +1,27 @@
+import React from "react";
+import { useState } from "react";
 import { Routes, Route } from "react-router-dom";
-import { Home, ProductList } from "./pages";
-import { Navbar, Footer } from "./components";
+import { Home, Profile, ProductList } from "./pages";
+import { Navbar, Footer, Login, Signup } from "./components";
 import Mockman from "mockman-js";
-import "./css/common.css";
 
 function App() {
+  const [isHome, setIsHome] = useState(true);
+
   return (
     <div className="App">
-      <Navbar />
+      <Routes>
+        <Route path="/" element={<Navbar isHome={true} />} />
+        <Route path="*" element={<Navbar isHome={false} />} />
+      </Routes>
```

Comment:
Can you explain this?

Reply:
I have used different classes for my navbar for different page renders, one contains absolute positioning as i needed my navbar to layer over my carousel, which was not required in other pages.
the above code sends a prop as true or false, which will tell my navbar if it is getting rendered in home page or anywhere else.

Reply:
So it is only home page where you want that style ?, if so then you can check the current path using the useLocation hook provided by react-router-dom and apply the class according to it